### ISSUE TITLE
[Feature Request] 

### ISSUE THREAD

**Issue by Awpmeister2170** [AdvancedPlugins/AdvancedMobs#67]
### Describe the bug

Basically this is the Mob below ( shadowguardian ) it was made via Modelengine / Blockbench and in the config you can write when it attacks etc as on Amobs currently the setting having hostile: true it doesnt do the Attack animation it only does a default attack and not the one made on Blockbench / Modelengine  but below it will allow the animations to be put in so the plugin knows what animation to use when its attacking 

shadowguardian:
  Type: SILVERFISH
  Health: 20
  Damage: 0
  Skills:
  
model{mid=shadowguardian} @Self ~onSpawn
state{mid=shadowguardian;s=attack;} @Self ~onAttack
state{mid=shadowguardian;s=chargedattack;} @Self ~onAttack
Options:
  Silent: true
  MovementSpeed: 0.1
  MaxCombatDistance: 25
  PreventOtherDrops: true
  PreventBlockInfection: true

### How to reproduce

N/A this is a Feature Request

See screenshot below on how Animations are named on Blockbench and how they are added above

### Screenshots / Videos

https://gyazo.com/0d3a2a24cdb03862ba7796c015cf9ec3

**Comment by aXe18** [comment]
Yeah also to this if possible add the feature from AdvancedItems that yo ucould choose the structures where mobs will spawn.
Like now you can choose biomes, but if structures could be added, it would be awesome to have certain mobs to spawn in different stuctures like ancient city
